controllers/admin/AdminMollieAjaxController.php

```diff
@@ -94,4 +98,9 @@ private function testApiKeys()
             ]
         ));
     }
+
+    private function closeUpgradeNotice()
+    {
+        Configuration::updateValue(Config::MODULE_UPGRADE_NOTICE_CLOSE_DATE, time());
```

Comment:
as I always mentioned - think always about interfaces and code testability . ClockInterface with method getNowTs() meaning gets current time as timestamp